tools/update_snapcraft.py

```diff
@@ -0,0 +1,105 @@
+"""Update the snapcraft yaml file.
```

Comment:
0 - yeah i faced that error also during testing, however since during CI, the script is only run once I left it as is since I thought running it manually a couple times was not a use case, but it should be easy to implement. will implement it in the update
1 - yeah i think its a good idea to have a tox target for this. I think it should be simple to include it here. will implement in the update
2 - python-tempestconf does not really have any official alignment with o7k releases that I can tell. however it has somewhat short list of requirements. i see no reason to always include the latest release.
3 - python-tempestconf is not included in the o7k release page and the releases repository. thats why we are checking the latest release from pypi.

Reply:
implemented 0 and 1 in the update.